src/include/sof/audio/buffer.h

```diff
@@ -144,6 +142,7 @@ struct comp_buffer {
 	uint32_t caps;
 	uint32_t core;
 	struct tr_ctx tctx;			/* trace settings */
+	bool is_shared;			/* buffer structure is shared between 2 cores */
```

Comment:
I see this checked in a couple of places, but I don't see it assigned anywhere?

Reply:
good catch, fixed